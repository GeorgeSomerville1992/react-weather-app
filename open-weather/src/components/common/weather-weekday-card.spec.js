import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import WeatherWeekDayCard from './weather-weekday-card';

describe('<WeatherWeekDayCard>', () => {
  it('renders structure', () => {
    const tree = renderer.create(
      <WeatherWeekDayCard
        currentDay={'Sunday'}
      />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})