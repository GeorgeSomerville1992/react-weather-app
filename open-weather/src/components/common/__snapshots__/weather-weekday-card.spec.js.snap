// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeatherWeekDayCard> renders structure 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1 sc-bdVaJa eqLehW"
>
  <div
    className="MuiCardMedia-root-29"
    src={undefined}
    style={
      Object {
        "backgroundImage": "url(\\"/static/images/cards/contemplative-reptile.jpg\\")",
      }
    }
    title="Contemplative Reptile"
  />
  <div
    className="MuiCardContent-root-31"
  >
    <h4
      className="MuiTypography-root-32 MuiTypography-body1-41 MuiTypography-gutterBottom-49"
    >
      Sunday
    </h4>
  </div>
</div>
`;
